src/figuresClasses.ts

```diff
@@ -1,19 +1,107 @@
-export interface Figure {
+type Shape ='triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: Shape;
```

Comment:
create an enum for shape and use it later like shape = Shape.rectangle; (instead of `type`)